scss/_tags.scss

```diff
@@ -0,0 +1,159 @@
+    margin: add(calc(-.5 * var(--#{$prefix}tag-icon-size)), var(--#{$prefix}tag-font-shift)) 0 calc(-.5 * var(--#{$prefix}tag-icon-size)) var(--#{$prefix}tag-icon-margin-start);
+  }
+
+  .close {
+    position: relative;
+    width: var(--#{$prefix}tag-close-size);
+    min-width: var(--#{$prefix}tag-close-size); // Here to avoid weird behavior on wrap
+    height: var(--#{$prefix}tag-close-size);
+    margin: add(calc(-.5 * var(--#{$prefix}tag-close-size)), var(--#{$prefix}tag-font-shift)) var(--#{$prefix}tag-close-margin-end) calc(-.5 * var(--#{$prefix}tag-close-size)) var(--#{$prefix}tag-close-margin-start);
+    color: inherit;
+    background-color: transparent;
+    border: 0;
+    outline-offset: 2px;
+    @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+
+    &::before {
+      position: absolute;
+      top: calc(-1 * var(--#{$prefix}tag-padding-y));
+      right: calc(-1 * var(--#{$prefix}tag-padding-y));
+      bottom: calc(-1 * var(--#{$prefix}tag-padding-y));
+      left: calc(-1 * var(--#{$prefix}tag-padding-y));
+      content: "";
+      @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+    }
+
+    &::after {
+      display: block;
+      height: 100%;
+      content: "";
+      background-color: currentcolor;
+      mask-image: escape-svg(var(--#{$boosted-prefix}close-icon));
+      mask-repeat: no-repeat;
+      mask-position: 50%;
+      mask-size: subtract(var(--#{$prefix}tag-close-size), .5rem);
+    }
+
+    &:hover,
+    &:active {
+      color: var(--#{$prefix}tag-active-color);
+      background-color: var(--#{$prefix}tag-active-decoration-color);
+    }
+
+    &:focus[data-focus-visible-added] {
+      outline: 3px solid var(--#{$prefix}tag-outline-color);
+      outline-offset: 2px;
+    }
+
+    &[disabled] {
+      pointer-events: none;
+    }
+  }
+
+  &[disabled],
+  &.disabled {
+    color: var(--#{$prefix}tag-disabled-color);
+    pointer-events: none;
+    border-color: var(--#{$prefix}tag-disabled-color);
+  }
+}
+
+// stylelint-disable selector-no-qualifying-type
+a.tag,
+button.tag,
+label.tag {
+  color: inherit;
+  text-decoration: none;
+  cursor: pointer;
+
+  &:hover {
+    border-color: var(--#{$prefix}tag-active-decoration-color);
+  }
+
+  &:focus[data-focus-visible-added] {
+    z-index: 3;
+    outline: .1875rem solid var(--#{$prefix}tag-outline-color);
+    outline-offset: .125rem;
+  }
+
+  &.active,
+  &:active {
+    color: var(--#{$prefix}tag-active-color);
+    background-color: var(--#{$prefix}tag-active-decoration-color);
+    border-color: var(--#{$prefix}tag-active-decoration-color);
+
+    &[disabled],
+    &.disabled {
+      background-color: var(--#{$prefix}tag-disabled-color);
+      border-color: var(--#{$prefix}tag-disabled-color);
+    }
+  }
+}
+// stylelint-enable selector-no-qualifying-type
+
+.tag-sm {
+  // scss-docs-start tag-dark-css-vars
```

Comment:
`// scss-docs-start tag-dark-css-vars` and `// scss-docs-start tag-sm-css-vars` are inverted